packages/core/tests/types/types.test.ts

```diff
@@ -64,7 +66,7 @@ class TestSourceCode
 	text: string;
 	ast: TestRootNode;
 	notMuch: "no" | false;
-	visitorKeys?: Record<string, string[]> | undefined;
+	visitorKeys?: Record<string, string[]>;
```

Comment:
TS option [`exactOptionalPropertyTypes`](https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes) forbids explicitly setting or initializing `visitorKeys` to `undefined`.